app-modules/meeting-center/src/Actions/GenerateEventRegistrationFormKitSchema.php

```diff
@@ -0,0 +1,91 @@
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AdvisingApp\MeetingCenter\Actions;
+
+use AdvisingApp\Form\Models\Submissible;
+use AdvisingApp\Form\Actions\GenerateFormKitSchema;
+
+class GenerateEventRegistrationFormKitSchema extends GenerateFormKitSchema
+{
+    public function __invoke(Submissible $submissible): array
+    {
+        $content = $this->generateContent($submissible);
+
+        $content = [
+            [
+                '$formkit' => 'radio',
+                'label' => 'Will you be attending?',
+                'id' => 'attending',
+                'name' => 'attending',
+                'options' => [
+                    [
+                        'label' => 'Yes',
+                        'value' => 'yes',
+                    ],
+                    [
+                        'label' => 'No',
+                        'value' => 'no',
+                    ],
+                ],
+                'validation' => 'required',
+            ],
+            [
+                '$el' => 'div',
+                'if' => '$get(attending).value === "yes"',
+                'children' => $content,
+            ],
+            [
+                '$formkit' => 'submit',
+                'if' => '$get(attending).value === "no"',
+                'label' => 'Submit',
+                'disabled' => '$get(form).state.valid !== true',
+            ],
+        ];
+
+        return [
+            '$cmp' => 'FormKit',
+            'props' => [
+                'type' => 'form',
+                'id' => 'form',
+                'onSubmit' => '$submitForm',
+                'plugins' => '$plugins',
+                'actions' => false,
+            ],
+            'children' => $content,
+        ];
```

Comment:
Could we reduce the use and reuse of `$content` here when it is only temporary? I understand maybe extracting the children out into a separate variable, but directly embedding `$this->generateContent($submissible)` into the static content could be fine?